### PR TITLE
test: passing eslint

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

File: test/helper.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 var assert = require('assert');
-var debug = require('debug')('strong-pm:test');
 var path = require('path');
 var util = require('util');
 
 require('shelljs/global');
 
+// XXX should be a way to ignore specific shelljs globals
```
Comment (rmg) [inline test/helper.js]: Are there so many that we can't just fix them?

Comment (rmg) [inline test/helper.js]: I added a fixup commit that removes this... it actually found a real leaked global on line 122!

**Comment by rmg** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by rmg** [comment]
My only comments were addressed by my own fixup.. so LGTM I guess?

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

File: test/helper.js (hunk 1 above)
Comment (sam-github) [inline test/helper.js]: I think I might have lost that fixup. do you still have it?

Comment (sam-github) [inline test/helper.js]: But note that what I really want is to get this onto master ASAP, so linting always happens on test, and this doesn't happen by accident: https://github.com/strongloop/strong-pm/blame/3bafbe266c800bec7c1203570bcfcd081615fec4/test/test-service-manager.js#L12-L13

(fixed in this branch)

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by rmg** [comment]
@sam-github my checkout was out of date, so I was able to reconstitute the shelljs fixup.

**Comment by rmg** [comment]
@slnode test please

File: test/test-pmctl-local.js [hunk 2]
```diff
@@ -40,7 +39,7 @@ helper.test('pmctl', function(t, pm) {
     t.expect(pmctl('restart', '1'), 'Service.*restarting');
     t.waiton(pmctl('get-process-count', '1'), /processes: 2/);
 
-    t.expect(pmctl('env-get', '1', '0'), 'No matching environment variables defined');
+    t.expect(pmctl('env-get', '1', '0'), 'No matching env variables defined');
```
Comment (rmg) [inline test/test-pmctl-local.js]: The second argument to `t.expect` is part of the assertion, it's not the test message.

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
CI is green :shipit: !

**Comment by sam-github** [comment]
YEAH!

**Pull request merged**